fastlane/Fastfile

```diff
@@ -49,11 +49,9 @@ platform :ios do
   end
 
   lane :create_simulator_build do
-    directory = prompt(text: "Please enter the path where we should output the sim build, relative to ./ios (i.e. ../my-simulator-build):")
```

Comment:
This fastlane lane is designed for humans to perform iOS releases. We use this argument in the current release process. If you want a lane without this arg, you should either create a different lane for whatever you need, or update the release process if you had a new one in mind.

Reply:
Seems like there's an better solution: the prompt method has a `:ci_input` argument, "The default text that will be used when being executed on a CI service".

Reply:
Awesome, I didn't know about `:ci_input`.